Doesn't build on Archlinux

```

arm-none-eabi-gcc  build/obj/crt0_v7m.o build/obj/chcoreasm_v7m.o  build/obj/crt1.o build/obj/vectors.o build/obj/chsys.o build/obj/chdebug.o build/obj/chvt.o build/obj/chschd.o build/obj/chthreads.o build/obj/chtm.o build/obj/chdynamic.o build/obj/chregistry.o build/obj/chsem.o build/obj/chmtx.o build/obj/chcond.o build/obj/chevents.o build/obj/chmsg.o build/obj/chmboxes.o build/obj/chqueues.o build/obj/chmemcore.o build/obj/chheap.o build/obj/chmempools.o build/obj/chcore.o build/obj/chcore_v7m.o build/obj/osal.o build/obj/hal.o build/obj/st.o build/obj/hal_queues.o build/obj/hal_mmcsd.o build/obj/can.o build/obj/i2c.o build/obj/icu.o build/obj/pal.o build/obj/serial.o build/obj/serial_usb.o build/obj/spi.o build/obj/usb.o build/obj/nvic.o build/obj/stm32_dma.o build/obj/hal_lld.o build/obj/st_lld.o build/obj/can_lld.o build/obj/pal_lld.o build/obj/i2c_lld.o build/obj/usb_lld.o build/obj/spi_lld.o build/obj/icu_lld.o build/obj/serial_lld.o build/obj/chprintf.o build/obj/syscalls.o build/obj/board.o build/obj/main.o build/obj/comm_usb_serial.o build/obj/irq_handlers.o build/obj/buffer.o build/obj/comm_usb.o build/obj/crc.o build/obj/digital_filter.o build/obj/ledpwm.o build/obj/mcpwm.o build/obj/servo_dec.o build/obj/utils.o build/obj/servo_simple.o build/obj/packet.o build/obj/terminal.o build/obj/conf_general.o build/obj/eeprom.o build/obj/commands.o build/obj/timeout.o build/obj/comm_can.o build/obj/ws2811.o build/obj/led_external.o build/obj/encoder.o build/obj/flash_helper.o build/obj/mc_interface.o build/obj/mcpwm_foc.o build/obj/gpdrive.o build/obj/confgenerator.o build/obj/timer.o build/obj/i2c_bb.o build/obj/virtual_motor.o build/obj/shutdown.o build/obj/hw.o build/obj/drv8301.o build/obj/drv8305.o build/obj/drv8320s.o build/obj/drv8323s.o build/obj/app.o build/obj/app_ppm.o build/obj/app_adc.o build/obj/app_sten.o build/obj/app_uartcomm.o build/obj/app_nunchuk.o build/obj/app_balance.o build/obj/app_custom.o build/obj/spi_sw.o build/obj/rf.o build/obj/rfhelp.o build/obj/nrf_driver.o build/obj/canard.o build/obj/canard_driver.o build/obj/esc_Status.o build/obj/esc_RawCommand.o build/obj/esc_RPMCommand.o build/obj/mpu9150.o build/obj/icm20948.o build/obj/ahrs.o build/obj/imu.o build/obj/bmi160.o build/obj/bmi160_wrapper.o build/obj/minilzo.o build/obj/bm_if.o build/obj/swdptap.o build/obj/timing.o build/obj/platform.o build/obj/exception.o build/obj/swdptap_generic.o build/obj/target.o build/obj/adiv5_swdp.o build/obj/adiv5.o build/obj/cortexm.o build/obj/nrf51.o build/obj/stm32f1.o build/obj/stm32f4.o build/obj/stm32h7.o build/obj/stm32l0.o build/obj/stm32l4.o build/obj/misc.o build/obj/stm32f4xx_adc.o build/obj/stm32f4xx_dma.o build/obj/stm32f4xx_exti.o build/obj/stm32f4xx_flash.o build/obj/stm32f4xx_rcc.o build/obj/stm32f4xx_syscfg.o build/obj/stm32f4xx_tim.o build/obj/stm32f4xx_iwdg.o build/obj/stm32f4xx_wwdg.o   -mcpu=cortex-m4 -O2 -ggdb -fomit-frame-pointer -falign-functions=16 -std=gnu99 -D_GNU_SOURCE -DBOARD_OTG_NOVBUSSENS  -fsingle-precision-constant -Wdouble-promotion -DUSE_STDPERIPH_DRIVER -ffunction-sections -fdata-sections -fno-common -mfloat-abi=hard -mfpu=fpv4-sp-d16 -fsingle-precision-constant -nostartfiles  -Wl,-Map=build/BLDC_4_ChibiOS.map,--cref,--no-warn-mismatch,--library-path=ChibiOS_3.0.2/os/common/ports/ARMCMx/compilers/GCC,--script=ld_eeprom_emu.ld,--gc-sections,--defsym=__process_stack_size__=0x800,--defsym=__main_stack_size__=0x800 -mno-thumb-interwork -mthumb  -lm -o build/BLDC_4_ChibiOS.elf
/usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/bin/ld: /usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/lib/thumb/v7e-m+fp/hard/libg.a(lib_a-abort.o): in function `abort':
abort.c:(.text.abort+0xa): undefined reference to `_exit'
/usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/bin/ld: /usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/lib/thumb/v7e-m+fp/hard/libg.a(lib_a-signalr.o): in function `_kill_r':
signalr.c:(.text._kill_r+0x12): undefined reference to `_kill'
/usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/bin/ld: /usr/lib/gcc/arm-none-eabi/9.3.0/../../../../arm-none-eabi/lib/thumb/v7e-m+fp/hard/libg.a(lib_a-signalr.o): in function `_getpid_r':
signalr.c:(.text._getpid_r+0x0): undefined reference to `_getpid'
collect2: error: ld returned 1 exit status
make: *** [ChibiOS_3.0.2/os/common/ports/ARMCMx/compilers/GCC/rules.mk:236: build/BLDC_4_ChibiOS.elf] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@
 ifeq ($(USE_OPT),)
   USE_OPT = -O2 -ggdb -fomit-frame-pointer -falign-functions=16 -std=gnu99 -D_GNU_SOURCE
   USE_OPT += -DBOARD_OTG_NOVBUSSENS $(build_args)
-  USE_OPT += -fsingle-precision-constant -Wdouble-promotion
+  USE_OPT += -fsingle-precision-constant -Wdouble-promotion -specs=nosys.specs
 endif
 
 # C specific options here (added to USE_OPT).

